recipe/meta.yaml

```diff
@@ -44,7 +48,16 @@ about:
     on the determinant of odd-ordered semimagic squares).  Some antimagic functionality
     is included.  The package also serves as a rebuttal to the often-heard comment "I
     thought R was just for statistics".
+  license_family: GPL2
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'  # [unix]
+  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-2'  # [win]
 
 extra:
   recipe-maintainers:
     - ocefpaf
```

Comment:
@cbrueffer thanks for taking care of these feedstocks! I really appreciate it.

If by any change I don't answer in those where I'm the only maintainer it is not b/c I ignored it, it is b/c I misse the ping :flushed: 

So please keep bugging me to merge whatever you need.

(I'm afraid that after some time away I may have lost of GH pings here and there.)

Reply:
No worries @ocefpaf , I have my personal commit robot in @bgruening :-D

Reply:
I'm getting a t-shirt made with the phrase: "I'm more than a merge button pusher."

I guess I need to get @bgruening one :smile: